src/main/webapp/WEB-INF/views/register.jsp

```diff
@@ -0,0 +1,18 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<style>
+    <%@include file='/WEB-INF/views/css/table_dark.css' %>
+</style>
+<html>
+<head>
+    <title>Register page</title>
+</head>
+<body>
+<h1 class="table_dark"> Welcome to our taxi family!</h1>
+<table class="table_dark">
+    <tr>
+        <th>To join us please contact our manager ;)</th>
+    </tr>
```

Comment:
ok, but let's add login and password to `/drivers/add` page
so users will be able to register themselves